traits-stubs/traits-stubs/tests/test_all.py

```diff
@@ -0,0 +1,28 @@
+# (C) Copyright 2005-2020 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+
+from pathlib import Path
+from unittest import TestCase
+
+import pkg_resources
+
+from util import MypyAssertions
```

Comment:
Can we replace this with the appropriately package-qualified import? Otherwise, it's hard to run the tests from outside this directory.